buddyboss/buddyboss-platform
# Fixed - Infinite loading after deleting messages

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1982.

### How to test the changes in this Pull Request:

1.Go to Messages
2. Message user
3. Click Delete your messages
4. Now compose screen should be displayed if no other thread found. or switch to the other thread if found.

### Proof Screenshots or Video
https://www.loom.com/share/097dfefbb76844938d90f77fddf81993



### Other information:
